src/applications/disability-benefits/all-claims/pages/addDisabilities.js

```diff
@@ -32,6 +32,14 @@ export const uiSchema = {
         {
           'ui:options': {
             freeInput: true,
+            inputTransformers: [
+              input => input.replace(/["”']/g, '’'),
+              input => input.replace(/[;–]/g, ' -- '),
+              input => input.replace(/[&]/g, ' and '),
+              input => input.replace(/[\\]/g, '/'),
+              input => input.replace(/([^a-zA-Z0-9\-’.,/() ]+)/g, ''),
+              input => input.replace(/\s{2,}/, ' '),
+            ],
```

Comment:
I'm not sure there's anything to test here, because this functionality in the field is tested, so we'd basically just be testing that `replace` works. Open to other ideas though.